### PR TITLE
Use jstransformer-scss and node-sass

### PR DESCRIPTION
**Pull request by RobLoach**
Build upon jstransformer-scss, brings in Boilerplate for updated metadata.

### CODE REVIEW

**Comment by ForbesLindesay** [comment]
Did this break the old test cases?

I think the scss module is native, so this may cause extra installation issues.  I wonder whether there should be two jstransformers modules for people to choose between

**Comment by RobLoach** [comment]
Interesting idea... Thinking about it, we could have as many modules as we wanted, and just inform it via the inputFormats. What would this package be for then?

**Comment by RobLoach** [comment]
To my knowledge, the [`sass`](https://www.npmjs.com/package/sass) package is deprecated. It...

- Lacks features from SASS and libsass
- Was put together by tj years ago; don't think anyone is willing to keep it going
- Has better implementations of a native JavaScript SASS-like in other systems like rework or postcss
- Confuses people as when people use "sass", they expect SASS... Not an incomplete implementation of it.

**Comment by tunnckoCore** [comment]
I dont see the problem?

**Comment by RobLoach** [comment]
> I dont see the problem? Let's merge.

We should replicate the currently existing test into a new folder, so rather than replacing it, we keep the existing one alive and well.

File: test/table/input.sass [hunk 1]
```diff
@@ -1,6 +1,6 @@
-+large
+=large
   :font-size 15px
-+striped
+=striped
   tr
     :background #fff
     +large
```
Comment (RobLoach) [inline test/table/input.sass]: The original test is now at `test/table/input.sass`.

**Comment by RobLoach** [comment]
The tests have been updated accordingly. Any thoughts?

**Pull request merged**